### ISSUE TITLE
Invalid in command-line window

### ISSUE THREAD

**Issue by horseinthesky** [folke/todo-comments.nvim#97]
When entering command-line window with `q:` get error:
![Screenshot_2](https://user-images.githubusercontent.com/19948672/158833645-825de023-2f46-44f1-a9ed-db2d7596d110.png)

**Comment by askfiy** [comment]
Me too, is your neovim version 0.7?

**Comment by horseinthesky** [comment]
> Me too, is your neovim version 0.7?

Yeap, I am using nightly. But I suppose behavior of command-line history window is pretty much the same as on stable release.

**Comment by rekaerst** [comment]
Before the pr is merged, you can use this temporary workaround
```lua
-- HACK: #104 Invalid in command-line window
local hl = require("todo-comments.highlight")
local highlight_win = hl.highlight_win
hl.highlight_win = function(win, force)
	pcall(highlight_win, win, force)
end

require("todo-comments").setup()
```

**Comment by horseinthesky** [comment]
It is not a big deal for me. I open command-line windows only by mistake :D

**Comment by vinnyA3** [comment]
can also use this fork - https://github.com/vinnyA3/todo-comments.nvim - until https://github.com/folke/todo-comments.nvim/pull/104 gets merged

**Comment by folke** [comment]
Should be fixed in #111
